test/e2e/framework/volume_util.go

```diff
@@ -85,8 +85,10 @@ type VolumeTestConfig struct {
 	// Wait for the pod to terminate successfully
 	// False indicates that the pod is long running
 	WaitForCompletion bool
-	// NodeName to run pod on.  Default is any node.
-	NodeName string
+	// ServerNodeName to run server pod on.  Default is any node.
```

Comment:
Maybe change the comment to say "NodeName". After reading this, I couldn't make the connection with the "nodeName" from pod spec right away.